### PR TITLE
Updated description of `use_ssh_args` option

### PR DESCRIPTION
**Pull request by rystraum**
This option has interactions with `ansible_ssh_common_args` as indicated in this issue thread: https://github.com/ansible/ansible/issues/16767

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: test/support/integration/plugins/modules/synchronize.py [hunk 1]
```diff
@@ -145,7 +145,7 @@
     default: yes
   use_ssh_args:
     description:
-      - Use the ssh_args specified in ansible.cfg.
+      - Use the ssh_args specified in ansible.cfg. Setting this to `yes` will also make `synchronize` use `ansible_ssh_common_args`.
```
Comment (bcoca) [inline test/support/integration/plugins/modules/synchronize.py:148]: i would just rephrase this to 'use the same options as the ssh connection plugin' as it also includes CLI and env vars.

**Comment by Akasurde** [comment]
Superseded by https://github.com/ansible-collections/ansible.posix/pull/97
